src/main/resources/templates/list-xbox.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://thymeleaf.org">
+<head>
+<meta charset="ISO-8859-1">
+<!-- Required meta tags -->
+<meta charset="utf-8">
+<meta name="viewport"
+	content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+<!-- Bootstrap CSS -->
+<link rel="stylesheet"
+	href="https://cdn.jsdelivr.net/npm/bootstrap@4.0.0/dist/css/bootstrap.min.css"
+	integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
+	crossorigin="anonymous">
+<title>Game List - Xbox</title>
+</head>
+<body>
+	<div class="container">
+		<h4>Xbox Games</h4>
+		<hr>
+		<table class="table table-bordered">
+			<thead>
+				<tr class="bg-success">
+					<th>GameName</th>
+					<th>Price</th>
+					<th>Console</th>
+				</tr>
+			</thead>
+			<tbody>
+				<tr th:each="xbox : ${xbox}">
+					<td th:text="${xbox.gameName}" />
+					<td th:text="${xbox.price}" />
+					<td th:text="${xbox.console}" />
+				</tr>
+			</tbody>
+		</table>
+	</div>
```

Comment:
I saw that you guys eventually intended to add an Xbox game list page, so I created one. I kept the code format / styling the same as your other pages for consistency.